aplt/scenarios.py

```diff
@@ -423,38 +423,47 @@ def loadtest():
         "aplt.scenarios:connect_and_idle_forever, %s, %s, %s"
         % (qty, stagger_delay, overall_delay))
 
+    # While not required, it's a good idea to label your arguments so
+    # you know how they're being applied by the function.
     yield spawn(
-        "aplt.scenarios:reconnect_forever, %s, %s, %s, %s, %s"
+        "aplt.scenarios:reconnect_forever, %s, %s, %s, "
+        "reconnect_delay=%s, run_once=%s"
         % (qty, stagger_delay, overall_delay, notif_delay, run_once))
 
     yield spawn(
-        "aplt.scenarios:register_forever, %s, %s, %s, %s, %s"
+        "aplt.scenarios:register_forever, %s, %s, %s, reg_delay=%s, "
+        "run_once=%s"
         % (qty, stagger_delay, overall_delay, notif_delay, run_once))
 
     yield spawn(
-        "aplt.scenarios:notification_forever, %s, %s, %s, %s, %s"
+        "aplt.scenarios:notification_forever, %s, %s, %s, notif_delay=%s, "
+        "run_once=%s"
         % (qty, stagger_delay, overall_delay, notif_delay, run_once))
 
     qty_stored = 30
     ttl = 300
-    flood_delay = 1
+    # flood_delay = 1
```

Comment:
let's just kill this line and fix the func docstring that confusingly references it